docker/mariadb/init.sql

```diff
@@ -0,0 +1,28 @@
+-- create metadata aspect table
```

Comment:
Curious if there is any diff between all 3 `init.sql`s ? If not, can we just have single `init.sql` shared between all dbs?

Reply:
Differences in column type

Reply:
I see, thanks @cobolbaby